src/QtUtils/CMakeLists.txt

```diff
@@ -12,17 +12,25 @@ target_link_libraries(QtUtils PUBLIC OrbitBase OrbitGl Qt5::Core CONAN_PKG::abse
 
 target_sources(QtUtils PUBLIC include/QtUtils/AssertNoQtLogWarnings.h
                               include/QtUtils/EventLoop.h
+                              include/QtUtils/ExecuteProcess.h
                               include/QtUtils/FutureWatcher.h
                               include/QtUtils/MainThreadExecutorImpl.h)
 
-target_sources(QtUtils PRIVATE FutureWatcher.cpp
+target_sources(QtUtils PRIVATE ExecuteProcess.cpp
+                               FutureWatcher.cpp
                                MainThreadExecutorImpl.cpp)
 
 set_target_properties(QtUtils PROPERTIES AUTOMOC ON)
 
+# Mock cli program
+add_executable(MockCliProgram)
+target_sources(MockCliProgram PRIVATE MockCliProgram/main.cpp)
+target_link_libraries(MockCliProgram CONAN_PKG::abseil)
```

Comment:
A mock keeps track of the methods that are called. This is more of a fake. See for example http://go/stadia-testing-best-practices.

Reply:
renamed it to FakeCliProgram